app/queries/servers/post.ts

```diff
@@ -85,3 +89,11 @@ export const queryRecentPostsInChannel = async (database: Database, channelId: s
     }
 };
 
+export function queryRecentMentions(database: Database) {
+    return queryRecentMentionIds(database).pipe(
+        switchMap((ids: string[]) => {
+            return database.get(POST).query(Q.where('id', Q.oneOf(ids))).fetch();
+        }),
+        catchError(() => of$([])),
```

Comment:
not needed as it will never trigger (catchError). Now see the comment on `queryRecentMentionIds`